src/pyetm/models/output_curves.py

```diff
@@ -147,20 +146,60 @@ def get_contents(
             self._merge_submodel_warnings(curve)
             return contents
 
-    def _find(self, curve_name: str) -> Optional[CarrierCurve]:
+    def get_curves_by_carrier_type(
+        self, scenario, carrier_type: str
+    ) -> dict[str, pd.DataFrame]:
+        """
+        Get all curves for a specific carrier type.
+
+        Args:
+            scenario: The scenario object
+            carrier_type: One of 'electricity', 'heat', 'hydrogen', 'methane'
+
+        Returns:
+            Dictionary mapping curve names to DataFrames
+        """
+        carrier_mapping = {
+            "electricity": ["merit_order", "electricity_price", "residual_load"],
+            "heat": [
+                "heat_network",
+                "agriculture_heat",
+                "household_heat",
+                "buildings_heat",
+            ],
+            "hydrogen": ["hydrogen", "hydrogen_integral_cost"],
+            "methane": ["network_gas"],
+        }
```

Comment:
This is a constant or config, we don't have to redefine it every me we hit this method. Let's find a better spot for it that's easier to access when we add or remove some downloads to etengine!